packages/theme/fonts/Sora/index.css

```diff
@@ -0,0 +1,8 @@
+@font-face {
+  font-family: "Sora";
+  src:
+    url('./Sora-VariableFont_wght.woff2') format('woff2'),
+    url('./Sora-VariableFont_wght.woff') format('woff'),
+    url('./Sora-VariableFont_wght.ttf') format("truetype-variations");
+  font-weight: 1 700 999;
```

Comment:
Could you explain why did you add it?
For me it looks unnecessary and breaking - Sora font is already defined at `theme/src/fonts/Sora` and could be imported as described at README. Woff2 & woff break variations, so only ttf is fine.

Reply:
I guess it is remnant after auto merging, I'll fix it